recipes/panda3d_viewer/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "panda3d_viewer" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/panda3d_viewer-{{ version }}.tar.gz
+  sha256: 623842303b9f568d80f5a300d35079738f578917916394abaca03ac4fa7c130a
+
+build:
+  number: 0
+  skip: True  # [win]
+  # The following skip will be removed in the feedstock
+  skip: True  # [py>36]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
```

Comment:
Just do ==3.6 and remove the skip for now.

Reply:
@ocefpaf As I mentioned in my first comment, we sure need the constaint python>=3.6 (i.e. skip when py<36), as a real constraint.
However, the build is ok for 3.6 alone, 3.7 alone... But they fail when they are build consecutively in the same container (proably due to cache file used by panda3d installer). I think this will not be a problem in the feedstock.
To be clean for the PR i finally did:
```
  skip: True  # [py<36]
  # The following skip will be removed in the feedstock
  skip: True  # [py>36]
```
to have a success build and just one line to remove once in the feedstock !